homeassistant/components/xiaomi_ble/strings.json

```diff
@@ -11,6 +11,9 @@
       "bluetooth_confirm": {
         "description": "[%key:component::bluetooth::config::step::bluetooth_confirm::description%]"
       },
+      "slow_confirm": {
+        "description": "There hasn't been a broadcast from this device in the last minute so we aren't sure if this device uses encryption or not. This may be because the device uses a slow broadcast interval. Confirm to add this device anyway, then the next time a broadcast is received you will be prompted to enter its bindkey if its needed."
```

Comment:
enter its bindkey if it's needed

Reply:
https://github.com/home-assistant/core/pull/76076